src/webui/www/private/download.html

```diff
@@ -191,6 +192,8 @@ <h2 class="vcenter">QBT_TR(Download Torrents from their URLs or Magnet links)QBT
             if (submitted)
                 window.parent.qBittorrent.Client.closeWindows();
         });
+
+        attachPathAutofill();
     </script>
 </body>
```

Comment:
@glassez 

Are you fine with attaching autocompletion this way?

Reply:
> @glassez
> 
> Are you fine with attaching autocompletion this way?

I have little to do with WebUI (client), so you'll have to wait for someone else's review.

Reply:
Maybe @Chocobo1 is a right person.